frontend/src/modules/app/ToolBar.tsx

```diff
@@ -161,16 +189,33 @@ export const ToolBar = ({
     string,
     Range
   >;
+  /**
+   * State to manage toggling ventilationState
+   */
   const [isVentilatorOn, setIsVentilatorOn] = React.useState(ventilating);
+  /**
+   * State to manage label for Landing page
+   */
   const [landingLabel, setLandingLabel] = useState('Loading...');
```

Comment:
@rohanpurohit Did you create this `useState` for `landingLabel`? If so, was it necessary to create a separate `useState` from `label`, or can they be merged?

Reply:
I think we need them to be seperate, as landing page needs to show 'Start' or 'Loading' instead of 'Start ventilation/Pause ventilation' or 'Loading' which would be the case elsewhere